scripts/create_dev_dbs.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/sh
```

Comment:
I think removing this file will break gitpod support; was it not working in the first place? Just want to make sure we're not removing existing functionality :)